### PR TITLE
Add ability for users to customise their install

### PR DESCRIPTION
**Pull request by Haegin**

### CODE REVIEW

File: profiles/dev/Brewfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+tap 'neovim/neovim'
```
Comment (Haegin) [inline profiles/dev/Brewfile]: kill this

File: profiles/dev/setup [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+function command_exists() {
+  command -v "$@" > /dev/null 2>&1
+}
+
+function install_lang() {
+  lang=$1
+  version=$2
+  echo "Installing $lang using ASDF"
+  if [[ -e $HOME/.asdf ]]; then
```
Comment (Haegin) [inline profiles/dev/setup:14]: need to install asdf first

Comment (Haegin) [reply profiles/dev/setup:14]: It's installed already by default.

File: profiles/dev/setup [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    source $HOME/.asdf/asdf.sh
+    asdf plugin-add $lang
+    latest=$(asdf list-all $lang | rg "^[0-9.]+$" | sort -n | tail -n 1)
+    asdf install $lang ${version:-$latest}
+  else
+    echo 'Could not install ${lang} as ASDF is not installed.'
+  fi
+}
+
+install_lang "elixir"
```
Comment (0xrohan10) [inline profiles/dev/setup:25]: Erlang needs to be installed.

**Review by 0xrohan10 — CHANGES_REQUESTED** [review]
A couple things

**Review by 0xrohan10 — APPROVED** [review]

**Pull request merged**